store/rootmulti/store_test.go

```diff
@@ -506,17 +508,15 @@ func getExpectedCommitID(store *Store, ver int64) types.CommitID {
 	}
 }
 
+// hashStores mirrors commitInfo.Hash and commitInfo.toMap but with a different input type
 func hashStores(stores map[types.StoreKey]types.CommitKVStore) []byte {
-	m := make(map[string][]byte, len(stores))
-	for key, store := range stores {
-		name := key.Name()
-		m[name] = storeInfo{
-			Name: name,
-			Core: storeCore{
-				CommitID: store.LastCommitID(),
-				// StoreType: store.GetStoreType(),
-			},
-		}.GetHash()
+	if len(stores) == 0 {
+		return nil
+	}
+	storeMap := make(map[string][]byte, len(stores))
+	for k, v := range stores {
+		storeMap[k.Name()] = v.LastCommitID().Hash
 	}
-	return SimpleHashFromMap(m)
+	rootHash, _, _ := merkle.SimpleProofsFromMap(storeMap)
```

Comment:
Why don't we just use `merkle.SimpleHashFromMap` here since we only need the hash and are discarding proofs.

Also, note that the functions `SimpleProofFromMap` and `SimpleHashFromMap` were removed in latest tendermint `master`. Maybe we want to add them back? cc: @alexanderbez